R/learner_gbm_regr_gbm.R

```diff
@@ -25,7 +25,25 @@
 #'
 #' @export
 #' @template seealso_learner
-#' @template example
+#' @examples
+#' if (requireNamespace("mboost", quietly = TRUE)) {
+#' # Define the Learner and set parameter values
+#'
+#'
+#' learner=lrn("regr.gamboost")
+#' print(learner)
+#'
+#' # Define a Task
+#' task = tsk("boston_housing")
+#'
+#'
+#' # Train the learner on the training ids
+#' learner$train(task)
+#'
+#' # print the model
+#' print(learner$model)
```

Comment:
why no pred?

Reply:
no suitable task found

Reply:
THis is also the wrong learner, it is the file for the gbm learner but the example shows gamboost.